lib/core-extensions/UserCSS/README.md

```diff
@@ -0,0 +1,89 @@
+# UserCSS extension
+
+A FreshRSS extension which give ability to create user-specific CSS rules to apply in addition of the actual theme.
+
+To use it, enable it on the extension panel in FreshRSS. You can add CSS rules by clicking on the manage button.
+
+## Changelog
+
+- 1.0.0 change rules location to user data folder; promoted to core extension
+- 0.0.1 initial version
+
+## Examples
```

Comment:
We could consider adding something in [our documentation](https://github.com/FreshRSS/FreshRSS/tree/edge/docs) for core extensions

Reply:
OK, let's move the description to that document.
Please let me know a few things.

1. Can I add it to [here](https://github.com/FreshRSS/FreshRSS/blob/edge/docs/en/users/05_Configuration.md) in the "Extensions" section?
2. Is it acceptable to add only English, because I cannot use French?
3. Can I reduce some of the configuration examples, as I don't think we need many of them?
4. Can I remove the README for this extension?

Reply:
I updated docs.